src/Sylius/Bundle/ThemeBundle/Repository/FailsafeThemeRepository.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Repository;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class FailsafeThemeRepository implements ThemeRepositoryInterface
+{
+    /**
+     * @var ThemeRepositoryInterface
+     */
+    private $unstableRepository;
+
+    /**
+     * @var ThemeRepositoryInterface
+     */
+    private $fallbackRepository;
```

Comment:
This is ok, but the unstable repo is damn funny name. Anyway, we need to somehow figure out how to get rid of these, cause catching all exceptions is pretty crazy. :D I am ok with figuring that later of course. :D